torchaudio/transforms.py

```diff
@@ -1025,3 +1025,127 @@ def forward(self, waveform: Tensor) -> Tensor:
             hp_lifter_freq=self.hp_lifter_freq,
             lp_lifter_freq=self.lp_lifter_freq,
         )
+
+
+class Synth(torch.nn.Module):
+    r"""This effect can be used to generate fixed or swept frequency audio tones with various wave shapes,
+     or to generate noise.
+
+    Args:
+        sample_rate (int, optional): Sample rate of audio signal (Default: ``16000``).
+        duration (float, optional): Duration in seconds of audio to synthesise (Default: ``1.0``).
+        wave_type (str, optional): is one of sine, square, triangle, sawtooth, trapezium, exp, white noise
```

Comment:
If the user initializes this with ```wave_type``` why then also have these as separate, public methods? Maybe you meant to make them private (mark with a prefix "_")?